Update coc documents to make the committee information up to date (#3225)
alan-turing-institute/the-turing-way: CODE_OF_CONDUCT.md

We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
Accordingly, everyone who participates in the Turing Way project is expected to show respect and courtesy to other community members at all times.

> *The Turing Way* Core Staff and our volunteer leaders, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.

Reviewer: I think there's something important about naming the leaders. Can we add me and @malvikasharan back in?

Also - is there a definition of core staff we can link to yet?

If not my preference would be to keep this as named leaders (K & M) but to incorporate this edit when we have more of the governance developed?

Author: ```suggestion
*The Turing Way* project leaders, Kirstie Whitaker and Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

Author: Added the names back in. Agree on linking to the governance documents when they are live in the book

> *The Turing Way* project leaders, Kirstie Whitaker and Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.

Author: ```suggestion
*The Turing Way* project leaders, Dr. Kirstie Whitaker and Dr. Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

To make clear what is expected, everyone participating in activities associated with the Turing Way project is required to conform to this Code of Conduct.
This Code of Conduct applies to all spaces managed by the Turing Way project including, but not limited to, in-person focus groups and workshops, and communications online via GitHub.

### Contact for Reporting

> The co-lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.

Reviewer: ```suggestion
The co-lead-investigators of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
```

> If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.

Reviewer: Is @annakrystalli still ok to do this? Very grateful if yes!

Author: confirmed - with a suggestion to look for an ombudsperson in 2024.

> If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.

Author: ```suggestion
If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.

Together they form the current Code of Conduct committee.
```

> You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.

Reviewer: Anonymously probably isn't the right word here if someone is reaching out to specific contact emails, right?

```suggestion
You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf, with or without your name associated with it.
```

Author: @aleesteele and @AlexandraAAJ can you please confirm that you understand your roles here?

Reviewer 2: This looks good to me.

Reviewer 3: Looks good to me too. Thanks @malvikasharan. Definitely would be interested in doing training regarding how to respond to CoC's going forward.

Author: Thanks, Anne. I suggest reading this for the role described here: https://malvikasharan.github.io/blogs/coc-facilitators/. I can look for my availability in the last quarter of the year for the Turing Way-specific training.

* excessive swearing
* incitement to violence, suicide, or self-harm
* continuing to initiate interaction (including photography or recording) with someone after being asked to stop
* publication of private communication without consent

## 2.3 Consequences of Unacceptable Behaviour

Participants who are asked to stop any inappropriate behaviour are expected to comply immediately.
This applies to all Turing Way community events and platforms, either online or in-person.
If a participant engages in behaviour that violates this Code of Conduct, any member of the core development team may warn the offender, ask them to leave the event or platform (without refund), or impose any other appropriate sanctions (see the [enforcement manual](#4-enforcement-manual) for details).

## 2.4 Feedback

This Code of Conduct is not intended as a static set of rules by which everyone must abide.
Rather, you are invited to make suggestions for updates or clarifications by contacting *The Turing Way* Core Staff Team at [turingway@turing.ac.uk](mailto:turingway@turing.ac.uk) or by making a pull request to this document on GitHub.

# 3 Incident Reporting Guidelines

## 3.1 Contact points

> If you feel able to, please contact Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - who are responsible for enforcing the Code of Conduct.

Author: ```suggestion
If you feel able to, please contact Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - who are responsible for enforcing the Code of Conduct.
```
